app/src/main/res/xml/home_preferences.xml

```diff
@@ -37,7 +37,8 @@
         android:dependency="@string/pref_key_pocket_homescreen_recommendations"
         android:layout="@layout/checkbox_left_sub_preference"
         android:key="@string/pref_key_pocket_sponsored_stories"
-        android:title="@string/customize_toggle_pocket_sponsored" />
+        android:title="@string/customize_toggle_pocket_sponsored"
+        app:isPreferenceVisible="false" />
```

Comment:
I am wondering if this would have an opposite effect for locales that should see this where the layout will now grow as the preference becomes visible. It's currently show only for US and CA locales https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt#37.

Reply:
@gabrielluong yes this is correct, good catch. I have moved the Preferences set up from onResume (Called when the fragment is visible to the user) to setPreferenceScreen (which is used to configure preferences prior to be being displayed)